test/fixtures/votes.yml

```diff
@@ -0,0 +1,17 @@
+vote1:
```

Comment:
This is a lot of work to get these user ids. Instead of using user_id and work_id in your yml, point directly to the user and work fixtures

```
vote1:
   user: raisah101
   work: forrest
```

Then in your tests you can make sure that the relation is set up and the foreign_keys are correct.